src/site/markdown/server/manage-webadmin.md

```diff
@@ -3048,6 +3049,40 @@ Response codes:
  - 204: Success (No content)
  - 400: Invalid request
  - 404: The mail queue does not exist
+ 
+### RabbitMQ republishing a mail queue from cassandra
```

Comment:
It doesn't make much sense as we don't republish the mail queue but the lost items.
What about "recover mails lost from RabbitMQ based on cassandra content"?